backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
The provided code patch introduces several issues with the user interface and functionality of the test result page. The original code had a clear distinction between different sections, but the updated version has merged these into a single page with unclear labels. This makes it difficult for users to navigate and understand what each section does.

A potential bug is that the button labels have been changed from "Test" to "Research", which may cause confusion among users who are expecting to test their input. Additionally, the placeholder text has been updated from "Enter your text here" to "Enter your topic here", but it would be more user-friendly to provide a clear indication of what type of input is expected.

Improvements could be made by restoring the original section labels and providing clearer instructions for users on how to use each section. Furthermore, the button labels should be updated to reflect their intended functionality, and the placeholder text should be adjusted accordingly.